airbyte-integrations/connectors/source-file-secure/source_file_secure/source.py

```diff
@@ -4,34 +4,19 @@
 
 import json
 import os
-import sys
+
+import source_file
 
 # some integration tests doesn't setup dependences from
 # requirements.txt file and Python can return a exception.
 # Thus we should to import this parent module manually
 from airbyte_cdk import AirbyteLogger
 from airbyte_cdk.models import ConnectorSpecification
 
-try:
-    import source_file.source
-except ModuleNotFoundError:
-    current_dir = os.path.dirname(os.path.abspath(__file__))
-    parent_source_local = os.path.join(current_dir, "../../source-file")
-    if os.path.isdir(parent_source_local):
-        sys.path.append(parent_source_local)
-    else:
-        raise RuntimeError("not found parent source folder")
-    import source_file.source
```

Comment:
This is not required anymore as the source file connector is installed as a normal python deps with pip install.